test/integration/util/integration_logger.go

```diff
@@ -0,0 +1,103 @@
+// +build integration
+
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"strings"
+)
+
+const (
+	messageInfoMaxLength = 7
+	timeHeaderLength     = 16
+)
+
+var (
+	IntegrationLog *log.Logger
+	logFile        *os.File
+)
+
+func init() {
+	IntegrationLog = log.New(ioutil.Discard, "", 0)
```

Comment:
Why would you need this. This means that an instance is created even before main is called. The actual log instance is created in StartLog, right - 'IntegrationLog = log.New(logFile, "", log.Ltime|log.Lmicroseconds)'. What's the purpose of this?